processor/configuration/src/main/resources/schemas/actions/google_pubsub_sink_0.1.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "gcp_project_id",
+    "gcp_destination_name",
+    "gcp_service_account_key"
+  ],
+  "properties": {
+    "gcp_project_id": {
+      "title": "Project Id",
+      "description": "The Google Cloud PubSub Project Id",
+      "type": "string"
+    },
+    "gcp_destination_name": {
+      "title": "Destination Name",
+      "description": "The Destination Name",
```

Comment:
Nitpick, it is not clear what value to actually put here.
From https://camel.apache.org/camel-kafka-connector/0.11.x/reference/connectors/camel-google-pubsub-kafka-sink-connector.html :
```
Required The Destination Name. For the consumer this will be the subscription name, while for the producer this will be the topic name.
```

Reply:
May be good to add that into our (and MC) description.

Reply:
I've added `Required The Destination Name. For the consumer this will be the subscription name, while for the producer this will be the topic name.` in the description